navfn/src/navtest.cpp

```diff
@@ -256,6 +256,13 @@ readPGM(const char *fname, int *width, int *height, bool raw)
 
   // set up cost map
   COSTTYPE *cmap = (COSTTYPE *)malloc(ncols*nrows*sizeof(COSTTYPE));
+  assert(cmap);
```

Comment:
Question:
Why do you need this "assert()".
Is it no problem to stop test program?

Reply:
Since it is an error process when memory for the file size could not be allocate, it is certainly not good to stop with assert.

Reply:
OK.